src/SelectMenu/SelectMenu.jsx

```diff
@@ -0,0 +1,70 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import Popover from '@material-ui/core/Popover'
+
+/**
+ * The `<SelectMenu>` component allows rendering a collection of `SelectMenuItem`s.
+ * This component uses MUI's `Popover`, accepting all of its props.
+ *
+ * ```js
+ * import { SelectMenu, SelectMenuItem } from '@theconversation/ui'
+ *
+ * <SelectMenu>
+ *   <SelectMenuItem onClick={() => select('Red')}>Red</option>
+ *   <SelectMenuItem onClick={() => select('Green')} selected>Green</option>
+ *   <SelectMenuItem onClick={() => select('Blue')}>Blue</option>
+ * </SelectMenu>
+ * ```
+ */
+export default function SelectMenu ({
+  anchorEl,
+  children,
+  open,
+  onClose,
+  ...other
+}) {
+  return (
+    <Popover
+      anchorEl={anchorEl}
+      onClose={onClose}
+      open={open}
+      anchorOrigin={{
+        vertical: 'bottom',
+        horizontal: 'left'
+      }}
+      transformOrigin={{
+        horizontal: 'left',
+        vertical: -5
+      }}
+      {...other}
+    >
+      {children}
+    </Popover>
+  )
+}
+
+SelectMenu.defaultProps = {
+  anchorEl: undefined
+}
+
+SelectMenu.propTypes = {
+  /**
```

Comment:
Would it make sense to add some demonstrations of the effects of changing these props (`anchorEl`, `onClose`, `open`) in the Usage stories?

Reply:
@benmacleod, `anchorEl`, `onClose` and `open` are required for the select menu to function. I added a couple of actions, so readers can better understand when those events and props are changed and called:

![example](https://user-images.githubusercontent.com/1538066/108500626-d4708b00-728e-11eb-9bc4-53ecf8b65b1a.gif)